wiki/release/release-process.md

```diff
@@ -454,7 +454,7 @@ Send out a PR request for review.
 
 ## Write release notes
 
-See [Pulsar Release Notes Guide](https://docs.google.com/document/d/1cwNkBefKyV6OPbEXnUrcCdVZi0i2BezqL6vAL7VqVC0/edit#).
+See [Pulsar Release Notes Guide](../release-note-guide.md).
```

Comment:
```suggestion
See [Pulsar Release Notes Guide](./release-note-guide.md).
```

Sorry. The first comment makes a mistake >_<

Reply:
No worries, I've updated it to `./release-note-guide.md`, which works well and is simple. Thank you!